tests/blockchain/eth/entities/actors/test_bidder.py

```diff
@@ -78,14 +124,14 @@ def test_claim(testerchain, agency, token_economics, test_registry):
     with pytest.raises(Bidder.BidderError):
```

Comment:
Should this be more specifically a `ClaimError`?

Reply:
✔️